### DISCUSSION TITLE
Default option for answering inline query

### DISCUSSION THREAD

**Discussion by AndreyDodonov-EH** [python-telegram-bot/python-telegram-bot#4309] [Q&A] [answered]
Is it possible to offer default result for inline query?
I.e. in  the [bot example](https://github.com/python-telegram-bot/python-telegram-bot/wiki/Extensions---Your-first-Bot) , the user still has to select Caps to "`upper`" their message.

Is it possible just to write
@<bot_name> <message>, and then hit Enter/Return and that's it, your message is processed and sent?

**Comment by Bibo-Joshi** [comment, marked as answer]
No, that's not possible. TG requires the user to choose an inline result

**Reply by AndreyDodonov-EH** [reply]
@Bibo-Joshi Appreciate the answer.
It's a pity though. 
Use-case which I had in mind is translation. 

Yes, there is already in Premium translation of the whole history, but it would be cool to send right away in the desired language.
Of course, menu can include either suggested translation or the list of languages to translate to, but still... every extra tap counts.
